contracts/base/ModuleManager.sol

```diff
@@ -3,8 +3,45 @@ pragma solidity >=0.7.0 <0.9.0;
 import {Enum} from "../libraries/Enum.sol";
 import {SelfAuthorized} from "../common/SelfAuthorized.sol";
 import {Executor} from "./Executor.sol";
-import {GuardManager, Guard} from "./GuardManager.sol";
 import {IModuleManager} from "../interfaces/IModuleManager.sol";
+import {IERC165} from "../interfaces/IERC165.sol";
+
+/// @title IModuleGuard Interface
```

Comment:
Please use consistent NatSpec comment style in the new code.

Reply:
https://github.com/safe-global/safe-smart-account/pull/758/commits/fad0ea2f0041e744c2ebdffdf5244794039bdbbb